server/graphql/v1/mutations/orders.js

```diff
@@ -766,16 +766,22 @@ export async function refundTransaction(_, args, req) {
   const transaction = await models.Transaction.findByPk(args.id, {
     include: [models.Order, models.PaymentMethod],
   });
+
   if (!transaction) {
     throw new errors.NotFound({ message: 'Transaction not found' });
   }
 
+  const collective = await models.Collective.findByPk(transaction.CollectiveId);
+  const isHost = await collective.isHost();
+  const HostCollectiveId = isHost ? collective.id : await collective.getHostCollectiveId();
```

Comment:
Is that implemented like something elsewhere? In which case would the `transaction.CollectiveId` be a host?

Reply:
Yeah in the case of adding fund manually, noticed that from inspecting this issue  https://github.com/opencollective/opencollective/issues/2042 locally. 

Also I think manual fund transactions should not have refund button shown, because I got the error below when I attempt to refund transaction created from "Add fund" form, was planning to open an issue regarding that.

```
 [GraphQL error]: Message: paymentMethod.refundTransaction is not a function,
Location: [object Object], Path: refundTransaction
```

Reply:
There is an issue for that I think (payment methods lacking refundTransaction).